client/deis.py

```diff
@@ -1348,6 +1335,27 @@ def _read_config_from_path(self, path):
 
         return env_dict
 
+    def _display_configuration(self, configuration, **kwargs):
+        oneline = kwargs.get('oneline', False)
+        highlight = kwargs.get('highlight', [])
+
+        keys = sorted(configuration)
+        if not oneline:
+            width = max(map(len, keys)) + 5
+            for k in keys:
+                k, v = encode(k), encode(configuration[k])
+                key = "{k:<" + str(width) + "}"
+                if k in highlight:
+                    self._logger.info(("\033[1m" + key + "\033[0m {v}").format(**locals()))
```

Comment:
let's use the `colored` library as we do here for better portability: https://github.com/deis/deis/blob/master/client/deis.py#L682-L691